app/views/templates/invoices/v4.slim

```diff
@@ -439,7 +439,7 @@ html
               = customer.city
           .body-2 = customer.state
           .body-2 = ISO3166::Country.new(customer.country)&.common_name
-          .body-2 = customer.email
+          .body-2 = customer.email.gsub(/,\s*/, ', ')
```

Comment:
Do we have invalid emails? If yes, should we fix them at DB level?

Reply:
Nope, DB attribute is valid. This is just formatting needed for HTML so that it can break this string to multiple lines correctly (upon space detection), when string is really long...